src/main/java/com/ioet/bpm/peopletime/timeevents/domain/TimeEvent.java

```diff
@@ -0,0 +1,59 @@
+package com.ioet.bpm.peopletime.timeevents.domain;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.*;
+import com.ioet.bpm.peopletime.timetemplates.domain.TimeTemplate;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import java.util.Date;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Data
+@DynamoDBTable(tableName = "people_time_event")
```

Comment:
Well, since this is the people_time service the table name should be people_time_time_event.

Reply:
Okay, I thought this would be too odd... But I'll change it!